debian/postinst

```diff
@@ -38,6 +38,9 @@ rsync -a /etc/dhcp/ ./etc/dhcp/
 sed -i 's@/bin/bash@/bin/sh@g' ./etc/passwd{,-}
 sed -i 's@/opt/delphix/server/bin/supportlogin@/bin/sh@g' ./etc/passwd{,-}
 
+mkdir -p ./opt/delphix/server/etc/{nginx.default,nginx}
+ln -s /opt/delphix/server/etc/nginx/nginx.conf ./etc/nginx/nginx.conf
```

Comment:
Just so I understand this properly, this symbolic link actually points to the `/opt/delphix/server/etc/nginx/nginx.conf` file that is within the recover image, is that right? Have you verified that cpio reproduces this as a symbolic link inside the image, instead of just copying the resolved file?

Reply:
Correct, and yes I have.